terraform/environments/analytical-platform-ingestion/dms/secrets.tf

```diff
@@ -18,6 +18,20 @@ module "cica_dms_tariff_database_credentials" {
     host     = "CHANGEME"
   })
 
+  create_policy       = true
+  block_public_policy = true
+  policy_statements = {
+    read = {
+      sid = "AllowDMSSourceRoleRead"
+      principals = [{
+        type        = "AWS"
+        identifiers = [module.cica_dms_tariff_dms_implementation.dms_source_role_arn]
+      }]
+      actions   = ["secretsmanager:GetSecretValue"]
+      resources = ["*"]
```

Comment:
Can this be scoped to a specific secret or prefix?

Reply:
This is being applied to the secret itself rather than the service accessing the secret, so it should already be restricted in scope to that particular secret. I don't think restricting this policy further will have any functional effect

Reply:
Ah my bad, I missed that!